pages/mesosphere/dcos/1.14/installing/production/deploying-dcos/installation/index.md

```diff
@@ -275,6 +275,15 @@ enable_ipv6: 'false'
     - 'foo.bar.com'
     - '.baz.com'
 
+[enterprise]
+## Create a bootstrap pre-shared key (Optional)
+[/enterprise]
+
+For additional security, create a random pre-shared key. This key will be used to authenticate requests very early in the installation process. 
+```bash
+mkdir genconf/ca
+cat /dev/urandom | tr -dc 'a-z' | fold -w 16 | head -n1 > genconf/psk
```

Comment:
Can we briefly say between which parts this is shared?

E.g.:
- bootstrap node
- DC/OS master nodes